modules/ROOT/pages/bloom-tutorial/scene-actions.adoc

```diff
@@ -7,6 +7,15 @@ _Scene actions_ are parameterized Cypher queries, much like xref:/bloom-visual-t
 The difference is that in a Scene action, the parameters can be the selected elements in your current selection instead of all available and matching elements in your graph.
 Also, whereas Scene actions are defined in the Perspective drawer under _Saved Cypher_, they are invoked from the context menu.
 
+[[write-transaction-action]]
+[IMPORTANT]
+====
+A Scene action can be sent as either a READ or a WRITE transaction to the database (provided you have appropriate permissions).
+In order to allow a WRITE transaction, you need to first enable this in the xref:bloom-visual-tour/settings-drawer.adoc[**Settings**] drawer.
```

Comment:
I had maybe replaced allow with enable or send here, allow sounds more like access control which is not really what this is about.

Reply:
Point taken! Thank you!